symengine/polys/uintpoly_flint.h

```diff
@@ -125,9 +125,9 @@ class UFlintPoly : public BaseType<Container, Poly>
         return r_iterator(this->template rcp_from_this_cast<Poly>(), -1);
     }
 
-    unsigned int size() const
+    int size() const
```

Comment:
@isuruf I don't think it can. Initially, it was giving an error such as this:
```
symengine/symengine/polys/upolybase.h:377:17: error:   overriding ‘int SymEngine::UPolyBase<Container, Poly>::size() const [with Container = SymEngine::fmpq_poly_wrapper; Poly = SymEngine::URatPolyFlint]’
```
It didn't make much sense to change the parent function. Hence it was changed here. What should be the approach here?